public/app/features/manage-dashboards/components/DashboardFileUpload.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC, FormEvent } from 'react';
+import { Forms, useTheme } from '@grafana/ui';
+
+interface Props {
+  onFileUpload: (event: FormEvent<HTMLInputElement>) => void;
+}
+
+export const DashboardFileUpload: FC<Props> = ({ onFileUpload }) => {
```

Comment:
I think this should be a `FileUpload` component that's imported from @grafana/ui

Reply:
Do you? We're currently using file upload at one place in grafana. I'd hold off on moving it until there are more use cases for it.

Reply:
kk